olive/evaluator/accuracy.py

```diff
@@ -62,6 +61,13 @@ def _metric_config_from_torch_metrics(cls):
     def _default_config(cls) -> Dict[str, ConfigParam]:
         return cls._metric_config_from_torch_metrics()
 
+    @staticmethod
+    def prepare_tensors(preds, target, dtypes=torch.int):
+        dtypes = dtypes if isinstance(dtypes, (list, tuple)) else [dtypes, dtypes]
```

Comment:
how about to add assertion:
assert len(dtypes) == 2

Reply:
done!